packages/gestalt/src/Box.js

```diff
@@ -883,8 +874,8 @@ const SizeDisplayPropType = PropTypes.oneOf([
   'inlineBlock',
 ]);
 
-// $FlowFixMe[prop-missing]
-Box.propTypes = {
+// $FlowFixMe
```

Comment:
What's the reason for this `$FlowFixMe`? I'm surprised it doesn't make you label the suppression type.

Reply:
Flow doesn't like the `propTypes` static on the `AbstractComponent` type (which makes sense). Feel like the easiest way to address this is #987 :D